src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/AddressLogEntry.orm.xml

```diff
@@ -28,5 +28,10 @@
             </options>
         </field>
         <field name="username" column="username" type="string" nullable="true" />
+
+        <indexes>
```

Comment:
to be rebased after merging and upmerging:

https://github.com/Sylius/Sylius/pull/16096